ddp-workspace/projects/ddp-sdk/src/lib/services/configuration.service.ts

```diff
@@ -58,4 +61,7 @@ export class ConfigurationService {
     scrollToErrorOffset = 100;
     defaultLanguageCode: string;
     rtlLanguages: string[] = [];
+    // The global flag indicates whether we need report JS errors to Google Cloud Error Report API
+    // It can be overridden for any single app (in proper app.module.ts)
+    doGcpErrorReporting = true;
```

Comment:
Can we make sure that we can set this configuration value by modifying the generated pepperConfig.js file?
Easiest way I can think of is by adding boolean configuration parameter to pepperConfig.js.ctmpl and reading in app.module.ts. Could also set `doGcpErrorReporting` when one of the two logging config parameters are missing. Up to you.
When running app locally most of the time we will not want to send errors to GCP. I could hand-edit this file, but git will identify it as a modified file. Rather edit pepperConfig.js that is not kept in the repo.
Take look at app.module.ts file for how we have been reading pepperConfig.js to set configuration values.

Reply:
Got it.
According to the point, it's fixed in 45027f3245726a8a1cc8213990df2a161f471922.